ui/components/app/advanced-gas-popover/advanced-gas-popover.js

```diff
@@ -0,0 +1,319 @@
+                </Typography>
+                <img height="18" src="./images/high-arrow.svg" alt="" />
+              </Box>
+              <Box
+                marginLeft={4}
+                className="advanced-gas-popover__input-subtext"
+              >
+                <Typography
+                  tag={TYPOGRAPHY.H8}
+                  variant={TYPOGRAPHY.H8}
+                  color={COLORS.UI4}
+                  fontWeight={FONT_WEIGHT.BOLD}
+                >
+                  <I18nValue messageKey="twelveHrTitle" />
+                </Typography>
+                <Typography
+                  tag={TYPOGRAPHY.H8}
+                  variant={TYPOGRAPHY.H8}
+                  color={COLORS.UI4}
+                >
+                  23-359 GWEI
+                </Typography>
+              </Box>
+            </Box>
+          }
+        />
+        <FormField
+          onChange={(value) => {
+            onManualChange?.();
+            setPriorityFee(value);
+          }}
+          titleFontSize={TYPOGRAPHY.H7}
+          titleText={t('priorityFee')}
+          titleUnit={t('gweiInParanthesis')}
+          tooltipText={t('advancedPriorityFeeToolTip')}
+          value={priorityFee}
+          detailText={priorityFeeFiatMessage}
+          numeric
+          bottomBorder
+          inputDetails={
+            <Box className="advanced-gas-popover__input-subtext">
+              <Box className="advanced-gas-popover__input-subtext">
+                <Typography
+                  tag={TYPOGRAPHY.H8}
+                  variant={TYPOGRAPHY.H8}
+                  color={COLORS.UI4}
+                  fontWeight={FONT_WEIGHT.BOLD}
+                >
+                  <I18nValue messageKey="currentTitle" />
+                </Typography>
+                <Typography
+                  tag={TYPOGRAPHY.Paragraph}
+                  variant={TYPOGRAPHY.H8}
+                  color={COLORS.UI4}
+                >
+                  1-18 GWEI
+                </Typography>
+                <img height="18" src="./images/low-arrow.svg" alt="" />
+              </Box>
+              <Box
+                marginLeft={4}
+                className="advanced-gas-popover__input-subtext"
+              >
+                <Typography
+                  tag={TYPOGRAPHY.H8}
+                  variant={TYPOGRAPHY.H8}
+                  color={COLORS.UI4}
+                  fontWeight={FONT_WEIGHT.BOLD}
+                >
+                  <I18nValue messageKey="twelveHrTitle" />
+                </Typography>
+                <Typography
+                  tag={TYPOGRAPHY.H8}
+                  variant={TYPOGRAPHY.H8}
+                  color={COLORS.UI4}
+                >
+                  0.1-127 GWEI
+                </Typography>
+              </Box>
+            </Box>
+          }
+        />
+      </Box>
+    </Popover>
+  );
+};
+
+AdvancedGasPopover.propTypes = {
+  onClose: PropTypes.func,
+};
+
```

Comment:
@NiranjanaBinoy : advance gas popup component will become very huge. Lets break this into smaller funtional components.

Reply:
I have moved the inputDetails to a functional component, now the component holds just 2 FormField components in a Box component.